andromedocs-theme/src/components/sidebar.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { Link, graphql, useStaticQuery } from "gatsby";
+import { Flex, Box, NavLink } from "theme-ui";
+
+const Sidebar = () => {
+  const { allSidebarYaml } = useStaticQuery(graphql`
+    query {
+      allSidebarYaml {
+        edges {
+          node {
+            label
+            link
+            items {
+              label
+              link
+            }
+          }
+        }
+      }
+    }
+  `);
+
+  console.log(allSidebarYaml);
+
+  return (
+    <Flex
+      as="nav"
+      sx={{
+        backgroundColor: "muted.4",
+        flexDirection: "column",
+        flex: "0 0 15%",
+        padding: 4,
+        "a:link": {
+          textDecoration: "none",
+        },
+        a: {
+          color: "inherit",
+        },
+      }}
+    >
+      {allSidebarYaml.edges.map(({ node: { label, link, items } }, index) => (
+        <NavLink key={index} sx={{}}>
+          <Box
+            sx={{
```

Comment:
How about adding a little padding here? `padding: 2` should suffice :pleading_face: